resources/js/Pages/Collections/View.tsx

```diff
@@ -90,7 +90,7 @@ const CollectionsView = ({
 
         return {
             traits: hasSelectedTraits ? selectedTraits : undefined,
-            owned: showOnlyOwned ? undefined : false,
+            owned: showOnlyOwned,
```

Comment:
With this adjustment it always add the `owned` param in the url, see video

I think that we should keep the default value as `undefined`, is not a breaking change per se and is not causing any issue but more like a personal preference since in this way we can prevent noise on the url, 



https://github.com/ArdentHQ/dashbrd/assets/17262776/f7d43b13-5ed5-49e6-afba-9bb94af40019

Reply:
this is needed after all to avoid the toggle from not hiding all NFTs when you own 0